ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/as-path.j2

```diff
@@ -0,0 +1,26 @@
+{% if as_path.access_lists is arista.avd.defined %}
+
+## AS Path Lists
+
+### AS Path Lists Summary
+{%     if as_path.regex_mode is defined %}
+
+AS Path Regex Mode is **{{ as_path.regex_mode }}**.
+{%     endif %}
+
+| List Name | Type | Match | Origin |
+| --------- | ---- | ----- | ------ |
+{%     for as_path_access_list in as_path.access_lists %}
```

Comment:
Use natural_sort filter. Also we can remove additional if check of that list as the filter will take care of that.
https://avd.sh/en/releases-v3.x.x/plugins/index.html#natural_sort-filter